src/tiled/tilesetmodel.cpp

```diff
@@ -22,17 +22,20 @@
 #include "tilesetmodel.h"
 
 #include "map.h"
+#include "relocatetile.h"
 #include "tile.h"
 #include "tiled.h"
 #include "tileset.h"
+#include "tilesetdocument.h"
 
 #include <QMimeData>
 
 using namespace Tiled;
 
-TilesetModel::TilesetModel(Tileset *tileset, QObject *parent):
+TilesetModel::TilesetModel(Tileset *tileset, TilesetDocument *tilesetDocument, QObject *parent):
```

Comment:
In case this `tileset` and `tilesetDocument` should always match, it would be better to just pass the `tilesetDocument`, since we can obtain the tileset from there.